app/models/setting/foreman_tasks.rb

```diff
@@ -13,7 +13,12 @@ def self.default_settings
           N_('Url pointing to the task troubleshooting documentation. '\
              'It should contain %{label} placeholder, that will be replaced with normalized task label '\
              '(restricted to only alphanumeric characters)). %{version} placeholder is also available.'),
-          nil)
+          nil),
+      set('foreman_tasks_polling_multiplier',
+          N_('Polling multiplier which is used to multiply the default polling intervals. '\
+             'This can be used to prevent polling too frequently for long running tasks.'),
+          1,
```

Comment:
maybe 1.0 to note user can use Float?

Reply:
I tried that, it gets truncated to an integer. Eseentialy it allows the user to input only non-negative integers.

Reply:
well, I'm ok with that. shortening intervals was not subject of this PR :)